haskeline.cabal

```diff
@@ -110,3 +111,62 @@ Library
         }
     }
     ghc-options: -Wall
+Executable haskeline-examples
+    hs-source-dirs:  examples
```

Comment:
Is there a way to make these `buildable: False` by default?  My goal would be to not install the example binaries in the user's `$PATH` with `cabal install haskeline`.